test/tests/bcs/function_dirichlet_bc/tests

```diff
@@ -1,29 +1,10 @@
 [Tests]
-  issues = ''
-  design = 'bcs/FunctionDirichletBC.md'
   [./test]
     type = 'Exodiff'
-    input = 'function_dirichlet_bc_test.i'
-    exodiff = 'out.e'
+    input = 'test.i'
+    exodiff = 'test_out.e'
+    issues = ''
```

Comment:
I'm not sure how to deal with this when it's before the github transition @permcody?

Reply:
I'll just move it back where it was, maybe it needs to be in the top block.

Reply:
Yeah, it's not letting a blank issues pass. What issues should I use? Here's the history
https://github.com/idaholab/moose/commits/next/framework/src/bcs/FunctionDirichletBC.C